### PR TITLE
Parser Benchmark

### PR DESCRIPTION
**Pull request by mofeiatwork**
Perform a benchmark of JSON parsers over vpack/rapidjson/simdjson, according to [issue107](https://github.com/arangodb/velocypack/issues/107).

### CODE REVIEW

**Comment by jsteemann** [comment]
@mofeiatwork : the results look very promising. Do you happen to know if the simdjson numbers also include building an internal data structure to access the parsed content later, or do they contain parsing only? Asking because the vpack parser does not only parse, but produces a ready to use velocypack as a side-effect. This is probably the most expensive part of invoking the velocypack parser. Anyway, the results look really good, so it's definitely worth to give simdjson a closer look.

File: tools/bench.cpp [hunk 1]
```diff
@@ -51,7 +58,7 @@ static void usage(char* argv[]) {
   std::cout << "out of cache. The target areas are also in a different memory"
             << std::endl;
   std::cout << "area for each copy." << std::endl;
-  std::cout << "TYPE must be either 'vpack' or 'rapidjson'." << std::endl;
+  std::cout << "TYPE must b: vpack/rapidjson/simdjson." << std::endl;
```
Comment (jsteemann) [inline tools/bench.cpp:61]:
```suggestion
  std::cout << "TYPE must be: vpack/rapidjson/simdjson." << std::endl;
```

**Review by jsteemann — APPROVED** [review]
:+1:

**Comment by jsteemann** [comment]
Hi @mofeiatwork : thanks for this awesome contribution!
I would love to merge your PR into the main branch, but can only do so if when a signed CLA is available. It seems that right now you are not in the list of contributors from which we have a signed CLA (https://github.com/arangodb/velocypack/pull/108#issuecomment-1013580482). In case you already sent it, please let me know.
Thanks!

**Comment by mofeiatwork** [comment]
@jsteemann I've sent the CLA email. It's my pleasure to contribute to the project.

**Comment by jsteemann** [comment]
@mofeiatwork : thanks a lot for your contribution!

**Pull request merged**